packages/render-html/package.json

```diff
@@ -89,7 +89,7 @@
   "dependencies": {
     "@jsamr/counter-style": "^2.0.2",
     "@jsamr/react-native-li": "^2.3.1",
-    "@native-html/transient-render-engine": "workspace:*",
+    "@native-html/transient-render-engine": "^11.2.3",
```

Comment:
This monorepo is meant to use Yarn workspaces; we cannot really avoid that. Having to manually handle versions would be cumbersome!

Reply:
Okay, I brought back previous implementation